server/api/impl/aws/provision_app.go

```diff
@@ -201,6 +187,24 @@ func (c *Client) createFunction(zipFile io.Reader, function, handler, runtime, r
 }
 
 // getFunctionName generates function name for a specific app
-func getFunctionName(appName, function string) string {
-	return fmt.Sprintf("%s_%s", appName, function)
+// name can be 64 characters long.
```

Comment:
I know I have asked this before but why not use content-based names (functions 64-byte SHAs)

Reply:
Because it will be harder for us to provision apps, until we automatize the provisioning. Semantic names are easier - No need to update manifest with SHA-names. After the changes in provisioner we can pivot to content-based names.
Basically this is for v1.